aula1/divisao.php

```diff
@@ -0,0 +1,158 @@
+                            $i,
+                            $i,
+                        );
+                    }
+                    ?>
+                </select>
+                <button type="submit">Ver</button>
+            </form>
+    
+            <div>
+                <?php
+                if ('' === $mensagemErro && null !== $tabuada && 0 !== $tabuada) {
+                    for ($j = 1; $j <= $valorMaximoTabuada; $j++) {
+                        $resultado = ($j * $tabuada) / $tabuada;
+    
+                        echo sprintf('<span>%d / %d = %d</span> </br>', ($j * $tabuada), $tabuada, $resultado);
+                    }
+                }
+    
+                if ('' !== $mensagemErro) {
+                    echo '<span>' . $mensagemErro . '</span>';
+                }
+                ?>
+            </div>
+        </section>
+
+        <section class="mt-5">
+            <button type="button" class="btn btn-success"><a href="index.php" class="text-white">Voltar para página anterior</a></button>
+        </section>
+    </main>
+
+    <footer id="contato" class="bg-success py-4">
+        <div class="container">
+            <div class="row text-center">
+                <div class="col-12 mb-3">
+                    <h2>Redes sociais</h2>
+                    <ul class="list-inline d-flex justify-content-center gap-4">
+                        <li class="list-inline-item">
+                            <a class="text-white text-decoration-none" href="https://github.com/JP-Pardinho">
+                                <i class="bi bi-github fs-4"></i>
+                                <span class="d-none d-sm-inline">GitHub</span>
+                            </a>
+                        </li>
+                        <li class="list-inline-item">
+                            <a class="text-white text-decoration-none" href="https://www.instagram.com/jp.pardinho/"
+                                target="_blank">
+                                <i class="bi bi-instagram fs-4"></i>
+                                <span class="d-none d-sm-inline">Instagram</span>
+                            </a>
+                        </li>
+                        <li class="list-inline-item">
+                            <a class="text-white text-decoration-none" href="https://www.linkedin.com/in/jppardinho/">
+                                <i class="bi bi-linkedin fs-4"></i>
+                                <span class="d-none d-sm-inline">LinkedIn</span>
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+
+                <div class="col-12">
+                    <p>
+                        Site criado por <a href="https://github.com/JP-Pardinho" target="_blank" class="text-white">
+                            JP-Pardinho</a> para o curso MS-Code.
+                    </p>
+                    <p>©2025 Loja do Pardin. Todos os direitos reservados.</p>
```

Comment:
para ficar o ano atual quando você voltar em 2026
```suggestion
                    <p>©2025 - <?= date('Y') ?> Loja do Pardin. Todos os direitos reservados.</p>
```